src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -269,6 +273,15 @@ function getPickedTokens(tokens: VotingPool['tokens']) {
             :pickedTokens="getPickedTokens(pool.tokens)"
           />
           <Ve8020Chip v-if="isVe8020Pool(pool)" />
+          <BalTooltip v-if="isGyro(pool)" :text="$t('clpTooltip')" width="56">
```

Comment:
We could extract this logic to its own component but I prefer to wait until we reuse it a third time.